### PR TITLE
Implement the Geek theme so that we can take advantage of their UI/UX patterns

### PR DESCRIPTION
**Pull request by edwinthinks**
### Describe your change in plain English.

This PR installs the https://geeksui.codescandy.com/ stylesheets and modifies a few pages as a short proof of concept. A few notable bits:
- Removes all custom CSS in favor of trying to keep it consistent with the theme (we can in a future PR just update styling later, imo)
- This doesn't include the JS from GeeksUI. I think we can handle that later if we need it
- Adds https://github.com/comfy/active_link_to which handles adding 'active' to nav links easily.

### Demo
Captura de pantalla 2023-09-13 a las 9 15 35
Captura de pantalla 2023-09-13 a las 9 15 39

### Link to the issue
N/A

### CODE REVIEW

**Comment by kasugaijin** [comment]
I love the look already. I also learned why coupling tests with DOM elements may not be ideal...makes front end changes harder to make.

**Comment by edwinthinks** [comment]
@kasugaijin all good :) -- I recorded a quick video of where the state of this PR is - https://jam.dev/c/d3d87a13-3add-423d-bdfb-2d97f51c9da5. In summary, I'am wanting this to be a foundational spot that we can start building ontop of. That being said there are some rough spots that we can address as we  develop further.

If this is looking good to you could you toss me an approve and we'll start picking parts to enhance?

**Review by kasugaijin — APPROVED** [review]
